src/cdk/drag-drop/drag-events.ts

```diff
@@ -71,3 +71,15 @@ export interface CdkDragMove<T = any> {
    */
   delta: {x: -1 | 0 | 1, y: -1 | 0 | 1};
 }
+
+/** Event emitted when the user swaps the position of two drag items. */
+export interface CdkDragSort<T = any, I = T> {
+  /** Index from which the item was sorted previously. */
+  previousIndex: number;
```

Comment:
Will this be the index since the last time a sort event or since the beginning of the drag?

Should be provide the `previousContainer` and `currentContainer`?

Reply:
It's since the last sort. I don't know how useful the `previousContainer` will be in this case and it'll add some complexity since we don't have it easily available in `_sortItem`.